osu.Game/OsuGame.cs

```diff
@@ -103,6 +104,9 @@ private Intro intro
 
         private readonly List<OverlayContainer> overlays = new List<OverlayContainer>();
 
+        private ExternalLinkOpener externalLinkOpener;
+        public void OpenUrlExternally(string url) => externalLinkOpener.OpenUrlExternally(url);
```

Comment:
Please put methods below the constructor and `load()` methods. Those that are currently not there are also misplaced but you don't have to change them for now.